articles/cms/wordpress/configuration.md

```diff
@@ -246,17 +273,15 @@ social Connections from working properly.
 
 * **Valid Proxy IP:** List the IP address of your proxy or load balancer to enable IP checks for logins and migration web services.
 
-* **Custom signup fields:** This field is the JSON that describes the custom signup fields for lock. It should be a valid json and allows the use of functions (for validation). [More info here](/libraries/lock/v10/new-features#custom-sign-up-fields).
-
-* **Extra settings:** A valid JSON object that includes options to call Lock with. This overrides all other options set above. For a list of available options, see [Lock: User configurable options](/libraries/lock/customization) (e.g.: `{"disableResetAction": true }`).
+* **Extra Settings:** A valid JSON object that includes options to call Lock with. This overrides all other options set above. For a list of available options, see [Lock: User configurable options](/libraries/lock/customization) (e.g.: `{"disableResetAction": true }`).
 
-* **Twitter consumer key and consumer secret:** The credentials from your Twitter app. For instructions on creating an app on Twitter, see [Obtain Consumer and Secret Keys for Twitter](/connections/social/twitter).
+* **Custom signup fields:** This field is the JSON that describes the custom signup fields for lock. It should be a valid json and allows the use of functions (for validation). [More info here](/libraries/lock/v10/new-features#custom-sign-up-fields).
```

Comment:
`* **Custom signup fields:** This field is the JSON that describes the custom signup fields for Lock. It should be valid JSON and allows the use of functions (for validation). You can find more information in the Lock documentation on [custom signup fields](/libraries/lock/v11/configuration#additionalsignupfields-array-).`

Changing a few copywriting items + the doc you were linking to is an old "what's new in Lock 10" doc, and could be a confusing one to be wandering around in + may be removed soon.